Template.php

```diff
@@ -3478,15 +3478,16 @@ public function set($par, $val) {
     if (($pos = $this->get_param_key((string) $par)) !== NULL) {
       return $this->param[$pos]->val = (string) $val;
     }
+    $p = new Parameter();
     if (isset($this->param[0])) {
-      $p = new Parameter();
       // Use second param as a template if present, in case first pair 
-      // is last1 = Smith | first1 = J.\n
-      $p->parse_text($this->param[isset($this->param[1]) ? 1 : 0]->parsed_text()); 
+      // is last1 = Smith | first1 = J.\m
+      $example = $this->param[isset($this->param[1]) ? 1 : 0]->parsed_text();
+      if (stripos($example, 'CITATION_BOT_PLACEHOLDER') !== FALSE) $example = '| param = val';
```

Comment:
Would 
```
$example = preg_replace(CITATION_BOT_PLACEHOLDER_BLAH, '', $example);
```
do a better job of keeping whatever format has already been used?  (Or could that just make things worse..?)

Reply:
Done.  Look at please.